quartz/components/Head.tsx

```diff
@@ -25,7 +25,9 @@ export default (() => {
         {cfg.baseUrl && <meta property="og:image" content={ogImagePath} />}
         <meta property="og:width" content="1200" />
         <meta property="og:height" content="675" />
+        <meta name="theme-color" content="#faf8f8" />
         <link rel="icon" href={iconPath} />
+        <link rel="manifest" href="/manifest.json" />
```

Comment:
use `baseDir` here, accessing something via absolute path to root is incorrect for any Quartz hosted on a subpath (e.g. `username.github.io/quartz`)